### PR TITLE
chore(requirements): reorganize and start depending on fixed werkzeug

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

**Comment by petrjasek** [comment]
not sure why you did this "refactoring", those libs should be set in setup.py, renaming requirements to requirements-dev is ok, but there should be no requirements file then

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+flake8
+nose
+pep8
+pyflakes<=1.1
+wooper
+
+-r requirements.txt
```
Comment (petrjasek) [inline requirements-dev.txt]: no need for this on travis, it installs the library when doing `pip install .`

**Comment by petrjasek** [comment]
now I see the issue, but imo it's problem with nginx config, not werkzeug

**Comment by actionless** [comment]
since werkzeug wasn't the issue i can remove it

but it still seems more readable and handy during the local development to have requirements separated from setup.py

**Comment by petrjasek** [comment]
dev-requirements are fine, but don't see why we should have requirements at all. you can use setup from dev-requirements via `-e .`

**Comment by actionless** [comment]
will it install package itself too or only its requirements?

**Comment by petrjasek** [comment]
both

**Pull request closed without merging**